### ISSUE TITLE
ability to add video the same way that image

### ISSUE THREAD

**Issue by eephyne** [neonelephantstudio/nemex#7]
For now we can only upload images.

But it could be nice to upload videos (even others kind of docs (like pdf or source code) could be useful)

**Comment by phoboslab** [comment]
I don't think it's feasible for this project to add video support. It essentially requires the ffmpeg binary and must allow very large POST sizes in the PHP config - which for most users means that they need a root server.

**Comment by heroheman** [comment]
Hmm - i think this is getting out of hand - there is no need to add an php encoder to a nemex site - I think what @eephyne (and me) are talking about is to add a movie file like an image - in other words: upload a movie (mp4, webm) like an image and just render the browser html5 video player / add the file with the video tag around it. 
Imho there is no need to provide it in different formats.

**Comment by eephyne** [comment]
Yes that's my opinion too (even if video processing can be a good idea too for video unsupported).

but @phoboslab got it right about the large POST sizes that can't be changed in shared web service.

**Comment by heroheman** [comment]
maybe this could be a kind of experiment flag in the config? for those uberspace users (like me) ;)
